assist_microphone/config.yaml

```diff
@@ -0,0 +1,29 @@
+---
+version: 1.0.0
+slug: assist_microphone
+name: Assist Microphone
+description: Use Assist with local microphone
+url: https://github.com/home-assistant/addons/blob/master/assist_microphone
+hassio_api: true
+arch:
+  - amd64
+  - aarch64
+  - armv7
+init: false
+discovery:
+  - wyoming
+map:
+  - share:ro
+options:
+  awake_wav: "/usr/src/sounds/awake.wav"
+  done_wav: "/usr/src/sounds/done.wav"
+  snd_volume_multiplier: 1.0
+  debug_logging: false
+schema:
+  awake_wav: str
+  done_wav: str
+  snd_volume_multiplier: float
+  debug_logging: bool
```

Comment:
Should we make a debug level option instead and tie it in with that?

Reply:
Can you go into more detail about what you mean here?

Reply:
Let's keep it at this for now, and ship this. We can always improve this part if needed.

👍